### PR TITLE
added example

### PR DESCRIPTION
**Pull request by trace2798**
added nextjs appdir todo redis example

### CODE REVIEW

File: examples/nextjs-appdir-todo-app-with-redis/tailwind.config.js
Comment (chronark) [inline examples/nextjs-appdir-todo-app-with-redis/tailwind.config.js:1]: This one is probably a duplicate, right?

File: examples/nextjs-appdir-todo-app-with-redis/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Next.js Todo App with Redis USING APP DIRECTORY
+products: ["redis"]
+stack: ["Next.js", "Typescript"]
+use_cases: ["Efficiently manage your Task"]
```
Comment (chronark) [inline examples/nextjs-appdir-todo-app-with-redis/README.md]:
```suggestion
use_cases: ["Primary Database"]
```

File: examples/nextjs-appdir-todo-app-with-redis/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title: Next.js Todo App with Redis USING APP DIRECTORY
```
Comment (chronark) [inline examples/nextjs-appdir-todo-app-with-redis/README.md]:
```suggestion
title: Next.js Todo App with Redis and app directory
```

**Review by chronark — CHANGES_REQUESTED** [review]
Looks really good, just a few small changes and we can merge :)

File: examples/nextjs-appdir-todo-app-with-redis/tailwind.config.js
Comment (trace2798) [reply examples/nextjs-appdir-todo-app-with-redis/tailwind.config.js:1]: My bad I did not notice it.

**Pull request merged**